### PR TITLE
Setup Ingress for Person service

### PR DESCRIPTION
**Pull request by taomoh**

### CODE REVIEW

File: helm/values/dev-values.yaml [hunk 1]
```diff
@@ -22,3 +22,10 @@ accruals:
     name: callisto-accruals-restapi
     port: 8080
   host: accruals.dev.callisto-notprod.homeoffice.gov.uk
+
+person:
+  name: callisto-person-restapi
+  service:
+    name: callisto-person-restapi
+    port: 8080
```
Comment (joannafawl) [inline helm/values/dev-values.yaml]: Is the port set to 9090 in person service?

Comment (JWeston-HO) [inline helm/values/dev-values.yaml]: Should this port be the same as accruals? 
Also applies to prod and test values

File: helm/values/test-values.yaml [hunk 2]
```diff
@@ -22,3 +22,10 @@ accruals:
     name: callisto-accruals-restapi
     port: 8080
   host: accruals.test.callisto-notprod.homeoffice.gov.uk
+
+peron:
+  name: callisto-peron-restapi
+  service:
+    name: callisto-peron-restapi
+    port: 8080
+  host: peron.test.callisto-notprod.homeoffice.gov.uk
```
Comment (joannafawl) [inline helm/values/test-values.yaml]: Typo for person

**Review by JWeston-HO — COMMENTED** [review]
Small comment. But looks good

File: helm/values/dev-values.yaml (hunk 1 above)
Comment (taomoh) [reply helm/values/dev-values.yaml]: Correct. Amended

Comment (taomoh) [reply helm/values/dev-values.yaml]: I think this should align with what's defined in Person service code as per Joanna's comment above.
Different ingress hosts can have the same port number. https://stackoverflow.com/a/62735312/2219361

File: helm/values/test-values.yaml (hunk 2 above)
Comment (taomoh) [reply helm/values/test-values.yaml]: Well spotted! FIxed

Comment (joannafawl) [reply helm/values/test-values.yaml]: There are a couple more in that section

Comment (taomoh) [reply helm/values/test-values.yaml]: Sorry. Fixed them now

**Review by joannafawl — APPROVED** [review]

**Review by JWeston-HO — APPROVED** [review]

**Pull request merged**